client.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include "libs/headers/bank.h"
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <semaphore.h>
+#include <string>
+#include <vector>
+#include <algorithm>
+
+void BankAction(const std::string& cmd, Bank*& bnk)
+{
+	static const std::vector<std::string> commands = 
```

Comment:
Use enums or store commands as constants.
For example: static const std::string GET_MAX{"get_max"};